kernel/build/config-amd64

```diff
@@ -635,7 +635,8 @@ CONFIG_CPU_FREQ_GOV_SCHEDUTIL=y
 #
 CONFIG_X86_INTEL_PSTATE=y
 # CONFIG_X86_PCC_CPUFREQ is not set
-# CONFIG_X86_AMD_PSTATE is not set
+CONFIG_X86_AMD_PSTATE=y
+CONFIG_X86_AMD_PSTATE_DEFAULT_MODE=3
```

Comment:
not sure if we should set this to active or passive?

Reply:
`CONFIG_X86_AMD_PSTATE_DEFAULT_MODE=3` (passive) is the default because it most closely matches the performance of `acpi-cpufreq`, which is used if amd-pstate is disabled. It feels appropriate to keep that as the default and require the user to select active or guided (e.g. `amd-pstate-epp`) and configure it.